internals/models.py

```diff
@@ -972,113 +974,135 @@ def put(self, notify=True, **kwargs):
-  intent_to_ship_url = db.LinkProperty()
-  ready_for_trial_url = db.LinkProperty()
-  intent_to_experiment_url = db.LinkProperty()
-  i2e_lgtms = db.ListProperty(db.Email)  # Currently, only one is needed.
-  i2s_lgtms = db.ListProperty(db.Email)
+  # intent_to_implement_url = db.LinkProperty()
+  intent_to_implement_url = ndb.StringProperty()
+  # intent_to_ship_url = db.LinkProperty()
+  # ready_for_trial_url = db.LinkProperty()
+  # intent_to_experiment_url = db.LinkProperty()
+  # i2e_lgtms = db.ListProperty(db.Email)  # Currently, only one is needed.
+  # i2s_lgtms = db.ListProperty(db.Email)
+  intent_to_ship_url = ndb.StringProperty()
+  ready_for_trial_url = ndb.StringProperty()
+  intent_to_experiment_url = ndb.StringProperty()
+  i2e_lgtms = ndb.StringProperty(repeated=True)  # Currently, only one is needed.
+  i2s_lgtms = ndb.StringProperty(repeated=True)
 
   # Chromium details.
-  bug_url = db.LinkProperty()
-  launch_bug_url = db.LinkProperty()
-  initial_public_proposal_url = db.LinkProperty()
-  blink_components = db.StringListProperty(required=True, default=[BlinkComponent.DEFAULT_COMPONENT])
-  devrel = db.ListProperty(db.Email)
-
-  impl_status_chrome = db.IntegerProperty(required=True)
-  shipped_milestone = db.IntegerProperty()
-  shipped_android_milestone = db.IntegerProperty()
-  shipped_ios_milestone = db.IntegerProperty()
-  shipped_webview_milestone = db.IntegerProperty()
+  # bug_url = db.LinkProperty()
+  # launch_bug_url = db.LinkProperty()
+  # initial_public_proposal_url = db.LinkProperty()
+  bug_url = ndb.StringProperty()
+  launch_bug_url = ndb.StringProperty()
+  initial_public_proposal_url = ndb.StringProperty()
+  # blink_components = db.StringListProperty(required=True, default=[BlinkComponent.DEFAULT_COMPONENT])
+  blink_components = ndb.StringProperty(repeated=True)
+  # devrel = db.ListProperty(db.Email)
+  devrel = ndb.StringProperty(repeated=True)
+
+  impl_status_chrome = ndb.IntegerProperty(required=True)
+  shipped_milestone = ndb.IntegerProperty()
+  shipped_android_milestone = ndb.IntegerProperty()
+  shipped_ios_milestone = ndb.IntegerProperty()
+  shipped_webview_milestone = ndb.IntegerProperty()
 
   # DevTrial details.
-  devtrial_instructions = db.LinkProperty()
-  flag_name = db.StringProperty()
-  interop_compat_risks = db.StringProperty(multiline=True)
-  ergonomics_risks = db.StringProperty(multiline=True)
-  activation_risks = db.StringProperty(multiline=True)
-  security_risks = db.StringProperty(multiline=True)
-  debuggability = db.StringProperty(multiline=True)
-  all_platforms = db.BooleanProperty()
-  all_platforms_descr = db.StringProperty(multiline=True)
-  wpt = db.BooleanProperty()
-  wpt_descr = db.StringProperty(multiline=True)
-
-  visibility = db.IntegerProperty(required=False)  # Deprecated
+  # devtrial_instructions = db.LinkProperty()
+  devtrial_instructions = ndb.StringProperty()
+  flag_name = ndb.StringProperty()
+  interop_compat_risks = ndb.StringProperty()
```

Comment:
The argument `multiline` has been deprecated as `StringProperty()` can contain strings with `\n` character.